src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -0,0 +1,236 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
+    SemidiscretizationHyperbolicParabolic
```

Comment:
Would it make sense to create a `SemidiscretizationParabolic` on its own and let a `SemidiscretizationHyperbolicParabolic` just wrap a `SemidiscretizationHyperbolic` and a `SemidiscretizationParabolic`?

Reply:
I was thinking it should just be a single semidiscretization since there's only one initial condition shared between the two (this approach treats the parabolic terms as secondary to the hyperbolic terms).

I thought it might also be helpful to have the combined semidiscretization to treat boundary conditions which involve both the hyperbolic and parabolic parts (e.g., `dot(advection_vector, normal) - diffusivity * normal_derivative = ...`).

Reply:
Another (possible) advantage of Jesse's approach is that it allows to take shortcuts and/or implement special behavior for the parabolic system. Since we failed to come up with a clean solution for BCs in #378, I propose to move ahead for now as it is and see where it takes us.

Reply:
Sounds good :+1: